common/persistence/src/main/java/app/coronawarn/server/common/persistence/domain/DiagnosisKey.java

```diff
@@ -45,6 +45,12 @@
  */
 @Entity
 @Table(name = "diagnosis_key")
+@NamedNativeQuery(
+    name = "DiagnosisKey.saveAllDoNothingOnConflict",
+    query = "INSERT INTO diagnosis_key"
```

Comment:
Just a note: Make sure to test on H2, also.

Reply:
✅